operators/multitenancy-operator/MultitenancyBindService.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const logger = require('../../common/logger');
+const utils = require('../../common/utils');
+const CONST = require('../../common/constants');
+const catalog = require('../../common/models').catalog;
+const BaseService = require('../BaseService');
+const bosh = require('../../data-access-layer/bosh');
+const MultitenancyAgent = require('./MultitenancyAgent');
+const eventmesh = require('../../data-access-layer/eventmesh');
+const assert = require('assert');
+
+class MultitenancyBindService extends BaseService {
+
+  constructor(guid, plan, parameters, bindResourceType, deploymentResourceType) {
+    super(plan);
+    this.guid = guid;
+    this.parameters = parameters;
+    this.director = bosh.director;
+    this.agent = new MultitenancyAgent(this.settings.agent);
```

Comment:
It would be good if we refrain from instantiating objects directly. Agent can be passed so that service class code can be reused.

Reply:
Thanks for the suggestion !
Let's take this change in next sprint.